### PR TITLE
Improve parse error reporting of apt and apama plugins

### PR DESCRIPTION
**Pull request by albinsuresh**
## Proposed changes

Print an informative error message and the usage when command-line parsing fails for apt and apama plugins.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [x] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [x] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: plugins/tedge_apama_plugin/src/main.rs [hunk 1]
```diff
@@ -302,7 +302,9 @@ fn main() {
     // On usage error, the process exits with a status code of 1
     let apama = match ApamaCli::try_parse() {
         Ok(apamacli) => apamacli,
-        Err(_e) => {
+        Err(err) => {
+            eprintln!("ERROR: {}", err);
+            ApamaCli::command().print_help().unwrap();
```
Comment (PradeepKiruvale) [inline plugins/tedge_apama_plugin/src/main.rs:307]: Remove unwrap

File: plugins/tedge_apt_plugin/src/main.rs [hunk 2]
```diff
@@ -238,7 +238,9 @@ fn main() {
 
     let apt = match AptCli::try_parse() {
         Ok(aptcli) => aptcli,
-        Err(_e) => {
+        Err(err) => {
+            eprintln!("ERROR: {}", err);
+            AptCli::command().print_help().unwrap();
```
Comment (PradeepKiruvale) [inline plugins/tedge_apt_plugin/src/main.rs:243]: Remove unwrap

Comment (albinsuresh) [reply plugins/tedge_apt_plugin/src/main.rs]: Replaced with `expect()` that prints a better error message than `unwrap()`.

**Review by PradeepKiruvale — APPROVED** [review]
LGTM

**Pull request merged**